websites/S/ShareX/presence.ts

```diff
@@ -0,0 +1,103 @@
+const presence = new Presence({
+  clientId: "888726220571811914" 
+}),
+ browsingStamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+  const showTimestamp: boolean = await presence.getSetting("timestamp"),
+  showButtons: boolean = await presence.getSetting("buttons"),
+presenceData: PresenceData = {
+  largeImageKey:
+    "sharex-logo",
+  smallImageKey:
+    "sharex-white-logo",
+  smallImageText: "Navigating on getsharex.com",
+  buttons: [
+  {
+    label: "View Page",
+    url: document.location.href
+  }
+]
```

Comment:
```suggestion
```

Reply:
Why do I have to remove buttons

Reply:
If you read guidelines
![Screenshot_2021-09-21-09-05-44-48.jpg](https://user-images.githubusercontent.com/54318514/134101065-ba352df5-80ce-458c-9b54-e44783d2a496.jpg)

Reply:
Didn't see that you use delete on main